apps/trading-e2e/src/integration/deposits.feature

```diff
@@ -0,0 +1,110 @@
+    And I approve the ethereum transaction
+    Then I can see the deposit is unsuccessful
+
+  @todo
+  Scenario: Cannot deposit if approved amount is lower than deposit amount
+    When I set "2" tokens to be approved
+    And I approve the asset tokens
+    And I can see the deposit form is displayed
+    And I select "" asset from the dropdown list
+    When I enter the following details
+      | Field         | Value    |
+      | To (Vega key) | xxxxxxxx |
+      | Amount        | 50       |
+    And I click to the deposit the funds
+    And I approve the ethereum transaction
+    Then I can see the deposit is unsuccessful
+    When I set "2" tokens to be approved
+    And i try to deposit "5" vega
+
+  @todo
+  Scenario: Can succesfully deposit (approved amount is greater than deposit)
+    When I set "200000000" tokens to be approved
+    And I approve the asset tokens
+    And I can see the deposit form is displayed
+    And I select "" asset from the dropdown list
+    When I enter the following details
+      | Field         | Value    |
+      | To (Vega key) | xxxxxxxx |
+      | Amount        | 50       |
+    And I click to the deposit the funds
+    And I approve the ethereum transaction
+    Then I can see the deposit is Successfull
+    And Balance is updated to reflect deposit amount
+
+  @todo
+  Scenario: Validation errors where no fields are filled in
+    When I submit a deposit form with empty fields
+    Then I can see validation errors present
+
+  @todo
+  Scenario: Deposit maximum amount you have in the wallet for a given asset
+    And I can see the deposit form is displayed
+    And I select "" asset from the dropdown list
+    When I enter the following details
+      | Field         | Value    |
+      | To (Vega key) | xxxxxxxx |
+      | Amount        | 0        |
+    When I click the use maximum button
+    Then I can see the field is updated with the maximum amount of the asset from my wallet
+
+  @todo
+  Scenario: Transaction fails if attempting to deposit more than available in wallet
+    And I can see the deposit form is displayed
+    And I select "" asset from the dropdown list
+    When I enter the following details
+      | Field         | Value    |
+      | To (Vega key) | xxxxxxxx |
+      | Amount        | 60000000 |
+    And I click to the deposit the funds
+    Then an error message is shown stating not enough tokens in wallet to deposit
+
+  @todo
+  Scenario: Deposit to a vega wallet key which is not your own
+    And I can see the deposit form is displayed
+    And I select "" asset from the dropdown list
+    When I enter the following details
+      | Field         | Value                      |
+      | To (Vega key) | VEGA KEY of another wallet |
+      | Amount        | 50                         |
+    And I click to the deposit the funds
+    And I approve the ethereum transaction
+    Then I can see the deposit is Successfull
+    And Balance is updated to reflect deposit amount
```

Comment:
Question non-blocking: I think we talked about just testing that we see confirmation that a deposit is in progress rather than waiting for the deposit to be complete